dotnet-fake.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <DotNetCliToolReference Include="dotnet-fake" Version="5.0.0-beta013" />
```

Comment:
why having a csproj ? Can't paket install this tool?

Reply:
I think there is still some work to do on the paket side and we need to publish fake as repotool